src/components/bookList.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import data from './bookData';
+import Options from './option';
+
+const BookList = () => (
+
+  <ul className="bookList">
+    {data.map((book) => (
+      <li
+        className="Book"
+        key={book.id}
+      >
+        <div>
+          <div>{book.category}</div>
+          <div>{book.title}</div>
+          <div>{book.author}</div>
+          <Options />
+        </div>
+        <div>
+          {book.percentage}
+          <p>Completed</p>
+        </div>
+        <div>
+          <p>CURRENT CHAPTER</p>
+          <div>{book.chapter}</div>
+          <button type="button">UPDATE PROGRESS</button>
+        </div>
+      </li>
+    ))}
+  </ul>
```

Comment:
According to the requirement, Kindly add the remove button 👍 

> Books - the default view
      Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no 
  funcionality yet)